course-contents.md

```diff
@@ -37,7 +38,7 @@
    - Reading data from existing sources
    - Creating data
    - Viewing data
-   - Basic knowledge about dataset (overall stats, data types)
+   - Basic knowledge about dataset (data types)
```

Comment:
```suggestion
   - Basic knowledge about dataset (overall stats, data types)
```

I'd keep this in here - I was planning to keep the .info() and .describe() methods in this section.